src/lib/parsers/Args.ts

```diff
@@ -204,7 +204,7 @@ export class Args {
 		const data = this.parser
 			.many()
 			.unwrapOr<Parameter[]>([])
-			.reduce((acc, parameter) => `${acc}${parameter.leading}${parameter.value}`, '');
+			.reduce((acc, parameter) => `${acc}${acc === '' ? '' : parameter.leading}${parameter.value}`, '');
```

Comment:
Much faster check:
```suggestion
			.reduce((acc, parameter, i) => `${acc}${i === 0 ? '' : parameter.leading}${parameter.value}`, '');
```

Although I'm going to make a PR to `@sapphire/lexure` to add a join function to do this, but more efficiently.

Reply:
Thanks for the feedback. I'll wait for [sapphiredev/utilities#458](https://github.com/sapphiredev/utilities/pull/458) to be merged and then update this to use the new join functions then.

Reply:
Updated in [a215cb0](https://github.com/sapphiredev/framework/pull/531/commits/a215cb064c94217dff5509af446b6828e8dc5db2).